sets.json

```diff
@@ -3277,145 +2807,5 @@
       "old_town_cross_444_720p50.y4m",
       "park_joy_444_720p50.y4m"
     ]
-  },
-  "scenecuts-720p": {
```

Comment:
same

Reply:
That set isn't very good, it's mostly the same 2-3 source videos, there are too many clips and not all of them have scene cuts, `vimeo-corpus-10s` was supposed to replace it.

Reply:
OK. I'm fine dropping it then.